### DISCUSSION TITLE
Beta 0.4体验

### DISCUSSION THREAD

**Discussion by TotoWang-hhh** [DearVa/ExplorerEx#53] [Ideas]
由于这个项目看上去有些时间没更新了，所以我也很久没关注了。

首先，如果这一版的发行还没附带环境的话，那是不是得考虑给软件瘦身一下，70多MB的安装包，下载的时候我也不知道里面有多少是运行环境。

![图片](https://user-images.githubusercontent.com/67090055/208066250-f5af8e52-4cf3-4f0b-b914-88f668575456.png)
![图片](https://user-images.githubusercontent.com/67090055/208086338-7ed2e131-2d34-4ff6-9a18-bf746836936f.png)

**Comment by DearVa** [comment]
附带环境的

**Reply by TotoWang-hhh** [reply]
好的，已经下不动很久了，现在刚刚刷新链接断点续传

**Comment by TotoWang-hhh** [comment]
小建议：可以去360提交一个人工审核，免费的，就直接点”误报反馈“就行了
![图片](https://user-images.githubusercontent.com/67090055/208098582-c2237a5d-9867-4cf3-9fd2-e831f62971af.png)

**Comment by TotoWang-hhh** [comment]
# 概括

设置总算有了，不过这一版给我的感觉就是：不是纯正的WinUI，而且窗口拖起来有点卡（不排除是我程序开多了）

# 设置

![图片](https://user-images.githubusercontent.com/67090055/208098978-2b502d2d-8e9e-4b72-99ea-e4937ccdb53c.png)

以及个人认为比较人性化的地方：

![图片](https://user-images.githubusercontent.com/67090055/208099840-416462cc-cf01-4e06-8161-c50d5dbb6d2e.png)

# 不纯正的WinUI

如果我没记错，正确的配色是背景：#0078DC（取色器告诉我正确的是0078D7），前景：白

![图片](https://user-images.githubusercontent.com/67090055/208099506-c4abf6c9-a270-49b5-aed3-422cfd4f9d35.png)

**Reply by DearVa** [reply]
好家伙，太细了

**Comment by TotoWang-hhh** [comment]
建议：
<del>希望文件区可以设置半透明，在背景的亚克力上面应该效果不错</del>
我刚发现其实是有这个选项的，但是调了似乎没用

以及希望默认文本编辑器可以增加一个”跟随系统“的选项，我的Win10强行装上Win11的UWP记事本之后这里选不了
![图片](https://user-images.githubusercontent.com/67090055/208100392-ea470c4f-fc52-4ad3-b15a-283d74fab789.png)

**Comment by TotoWang-hhh** [comment]
它刚刚好像自己闪退了，或者只是我不经意间把它关了
![图片](https://user-images.githubusercontent.com/67090055/208100748-d244c360-bc73-4a4a-9a11-49ce188674c1.png)

**Comment by TotoWang-hhh** [comment]
# 重命名/双击的判定方式问题
这个问题我会在issue中提出，包括前面说的”不纯正的WinUI“

**Comment by TotoWang-hhh** [comment]
印象极其良好的几个UWP软件（顺序按排名），或许您可以在界面设计上参考一下
![图片](https://user-images.githubusercontent.com/67090055/208105674-331da90c-6c56-4475-b24c-f20b6faf69c4.png)
![图片](https://user-images.githubusercontent.com/67090055/208106238-82219996-86dc-4e88-91ff-4f6e7f011c1b.png)
